notebooks/amazon_forest_notebook.py

```diff
@@ -313,21 +316,31 @@
 
 # <codecell>
 
-# For now we'll just put all thresholds to 0.2 but we need to calculate the real ones in the future
-#thresholds = [0.2] * len(labels_set)
+thresholds = [0.2] * len(labels_set)
 
+# <markdowncell>
 
-# <codecell>
+# Now lets map our predictions to their tags by using the thresholds
 
-thresholds = [0.24, 0.2, 0.2, 0.2, 0.2, 0.14, 0.05, 0.2, 0.2, 0.25, 0.25, 0.24, 0.2, 0.25, 0.2, 0.2, 0.25]
+# <codecell>
 
-# <markdowncell>
+def map_predictions(predictions, thresholds):
```

Comment:
Yea this is because this function and `predict` are the only two functions I am using from `keras_helper`. I am not sure we should still keep that file just for two functions?